MobileWallet/Common/Extensions/CGColor+App.swift

```diff
@@ -0,0 +1,55 @@
+//  CGColor+App.swift
```

Comment:
Move this file contents to Theme.swift. We're keeping all references to colors and fonts in one place there.